README.md

```diff
@@ -83,7 +83,8 @@ module "castai_gke_cluster" {
         }
         compute_optimized = false
         storage_optimized = false
-        architectures = ["amd64"]
+        is_gpu_only       = false
```

Comment:
Any reason why this API is now prefixed with `is`? 
All other are kinda nouns `storage, compute, spot` and this one is `is_stuff`? Bit fo new naming ?

Reply:
The only reason is to be consistent with the field naming in API. Not sure why it was named this way there but yes it does not seem consistent